packages/matter.js/src/device/AutoDevice.ts

```diff
@@ -87,7 +87,10 @@ export interface StateManager<State extends BaseState> {
 }
 
 type Constructor = new (...args: any[]) => any;
-type UntypedClusterServer = <() => >;
+
+// TODO - should this be properly typed and added to the public API?  I'm
```

Comment:
Yes should be typed ... but also "request" missing here or?

Reply:
The request body I had as a separate argument.  This is just internal implementation detail at the moment, there's proper typing in the public API.  I'll move development to a new branch so I don't slam you with updates every time I push a WIP. 😉